mlir/include/mlir/Dialect/AMDGPU/Utils/Chipset.h

```diff
@@ -47,6 +47,13 @@ struct Chipset {
   DEFINE_COMP_OPERATOR(>)
   DEFINE_COMP_OPERATOR(>=)
 #undef DEFINE_COMP_OPERATOR
+
+  bool isGfx940() const {
+    return majorVersion == 9 && minorVersion >= 4 && minorVersion < 5;
```

Comment:
The condition is weird, this should be just `minorVersion == 4`. Also, I'd call it something like `isGfx940Series`, since it's not a strict match.

Reply:
Faulty merge.  It made more sense before steppingVersion, as >=0x40 && < 0x50.